src/app/api/admin/common.js

```diff
@@ -54,6 +55,8 @@ export function transformServiceProvider(instance, modelName) {
   if (modelName === RESOURCES.organization) {
     // eslint-disable-next-line no-param-reassign
     instance.organizationTypesIds = instance.type.map(orgType => orgType.id);
+    // eslint-disable-next-line no-param-reassign
+    instance.expertSpecializationIds = instance.expertSpecializations.map(({ id }) => id);
```

Comment:
You can put
`/* eslint-disable no-param-reassign */`
at file start to disable this rule for the file.
Or put /* eslint-disable no-param-reassign */ and /* eslint-enable no-param-reassign */,  before and after the function to disable only specific ranges of the file.
To avoid separate disabling for each separate line,  because it is impossible to read the function.

And,  just FYI, it would be good to have a link to the GitHub issue to understand why we can't return a new obj,  but mutate the existing one, and maybe at some point in the future issue will be fixed and it will be easy to check

Reply:
as for why we can't return new object - that is mystery for me, i didn't find any related documentation, and in plugin we use i didn't find relevant documentation

but! i've fixed comments 🥳